Utils/hdinsight-node-common/resources/htmlResources/hdinsight/job/html/index.html

```diff
@@ -51,7 +51,7 @@
             <div></div>
                 <table id="myTable" class="table table-bordered table-condensed table-striped sortable ui-widget-content">
                     <thead class="ui-widget-content">
-                    <tr align="center" class="ui-widget-content">
+                    <tr align="center" class="ui-widget-content" style="word-wrap: break-word;">
```

Comment:
confirm with lakshmi about 
1870901

Reply:
confirmed, [1870900](https://dev.azure.com/mseng/VSJava/_workitems/edit/1870900) and [1870901](https://dev.azure.com/mseng/VSJava/_workitems/edit/1870901) are both overlap issue.